components/script/dom/audiotrack.rs

```diff
@@ -18,14 +20,17 @@ pub struct AudioTrack {
     label: DOMString,
     language: DOMString,
     enabled: Cell<bool>,
+    track_list: DomRefCell<Option<Dom<AudioTrackList>>>,
```

Comment:
Make that `Option<Dom<AudioTrackList>>`, you never actually need to mutate it. If I'm wrong about it never changing, it should be `MutNullableDom<AudioTrackList>`.